basic_system/src/system_implementation/system/public_input.rs

```diff
@@ -159,6 +162,8 @@ pub struct BatchOutput {
     pub upgrade_tx_hash: Bytes32,
     /// Linear keccak256 hash of interop roots
     pub interop_roots_rolling_hash: Bytes32,
+    /// New settlement layer chain id, if updated. 0 otherwise.
```

Comment:
Should we explicitly forbid setting it to 0?

Reply:
No longer applicable in the new version